### PR TITLE
docs: append INSTALL doc

### PR DESCRIPTION
**Pull request by hzhuang1**
Add instructions on building UADK and running test cases in INSTALL document.

Signed-off-by: Haojian Zhuang <haojian.zhuang@linaro.org>

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -0,0 +1,69 @@
+Deploying UADK
+
+UADK is a framekwork for user application to access hardware, and it depends
+UACCE in kernel.
+
+
+1. Kernel Requirement
+
+User needs to make sure that UACCE is already supported in Linux kernel. The
+kernel version should be at least v5.9.
```
Comment (wangzhou) [inline INSTALL:10]: In fact, we need a SVA kernel, so v5.9 is not enough :)

File: INSTALL [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+UACCE could be built as module or built-in.
+
+Here's an example to enable UACCE with hardware accelerator in Hisilicon
+platform.
+
+    CONFIG_UACCE=y
+    CONFIG_CRYPTO_DEV_HISI_SEC2=y
+    CONFIG_CRYPTO_DEV_HISI_QM=y
+    CONFIG_CRYPTO_DEV_HISI_ZIP=y
+    CONFIG_CRYPTO_DEV_HISI_HPRE=y
```
Comment (wangzhou) [inline INSTALL:24]: let's also mention SVA, SMMU, PCI_PASID related configures

File: INSTALL [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    Asynchronous mode
+    $ sudo ./test/hisi_zip_test/test_sva_perf -b 8192000 -l 1000 -v -m 1
+
+Test on sec hardware accelerator
+
+    Synchronous mode
+    $ sudo ./test/hisi_sec_test/test_hisi_sec -cipher 0 -optype 0 -pklen 16 -keylen 16 -times 1 -sync -multi 1
+    Asynchronous mode
+    $ sudo ./test/hisi_sec_test/test_hisi_sec -cipher 0 -optype 0 -pklen 16 -keylen 16 -times 1 -async -multi 1
+
```
Comment (wangzhou) [inline INSTALL:63]: let's also mention digest test case, in fact, just replace -cipher above to -digest

Comment (hzhuang1) [reply INSTALL:71]: fixed

File: INSTALL (hunk 2 above)
Comment (hzhuang1) [reply INSTALL:30]: fixed

File: INSTALL (hunk 1 above)
Comment (hzhuang1) [reply INSTALL]: fixed

**Pull request merged**